apps/api/src/graphql/schema/study-rooms.ts

```diff
@@ -1,4 +1,4 @@
-export const studyRoomsGraphQLSchema = `#graphql
+export const studyRoomsSchema = `#graphql
```

Comment:
This change is not within scope - consider creating a separate issue if refactoring this is necessary.